### PR TITLE
Add support for batch compute in calculators

### PR DESCRIPTION
**Pull request by maclandrol**
Checklist:

- [x] Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate).
- [x] Update the API documentation if a new function is added or an existing one is deleted.
- [x] Added a `news` entry.
  - _copy `news/TEMPLATE.rst` to `news/my-feature-or-branch.rst`) and edit it._

---

### CODE REVIEW

File: molfeat/calc/base.py [hunk 1]
```diff
@@ -30,7 +30,40 @@ def __init__(cls, name, bases, attrs):
 
 
 class SerializableCalculator(abc.ABC, metaclass=_CalculatorMeta):
-    """Interface to define a serializable calculator"""
+    """Interface to define a serializable calculator
+
+    ???+ tip "Subclassing SerializableCalculator"
```
Comment (hadim) [inline molfeat/calc/base.py:35]: `???+ tip` typo or legit?

**Review by hadim — APPROVED** [review]

File: molfeat/calc/base.py (hunk 1 above)
Comment (maclandrol) [reply molfeat/calc/base.py:35]: legit. Learned that yesterday

Comment (hadim) [reply molfeat/calc/base.py:35]: Nice looking

image

Comment (cwognum) [reply molfeat/calc/base.py:35]: FYI: https://jejjohnson.github.io/research_journal/resources/mkdocs/#extensions

**Pull request merged**